coc/client.py

```diff
@@ -538,12 +541,61 @@ async def get_warlog(
         if self.correct_tags:
             clan_tag = correct_tag(clan_tag)
 
+        if paginated and not kwargs.get("limit", None):
+            kwargs["limit"] = 5
         try:
-            data = await self.http.get_clan_warlog(clan_tag)
+            data = await self.http.get_clan_warlog(clan_tag, **kwargs)
         except Forbidden as exception:
             raise PrivateWarLog(exception.response, exception.reason) from exception
 
-        return [cls(data=wdata, client=self, **kwargs) for wdata in data.get("items", [])]
+        return ClanWarLog(data=data, client=self, cls=cls, clan_tag=clan_tag)
+
+    async def get_raidlog(
+        self,
+        clan_tag: str,
+        cls: Type[RaidLogEntry] = RaidLogEntry,
+        paginated: bool = True,
+        **kwargs
+    ) -> RaidLog:
+        """Retrieve a clan's raid log.
+        Set paginated = False to get the full raid log with one API call.
+
+        Parameters
+        -----------
+        clan_tag : str
+            The clan tag to search for.
+
+        Raises
+        ------
+        TypeError
+            The ``cls`` parameter must be a subclass of :class:`RaidLogEntry`.
+
+        NotFound
+            No clan was found with the supplied tag.
+
+        Maintenance
+            The API is currently in maintenance.
+
+        GatewayError
+            The API hit an unexpected gateway exception.
+
+
+        Returns
+        --------
```

Comment:
```suggestion

        Returns
        --------
```
Lines should have consistent spacing